Source/Resources/Content/MSGraph/DeviceAppManagement/IosManagedAppProtections/Protect Policy Managed Client Apps on Unmanaged iOS Devices.json

```diff
@@ -291,7 +291,7 @@
   "contactSyncBlocked": false,
   "customBrowserProtocol": "",
   "dataBackupBlocked": true,
-  "description": "Configures default behavior for policy managed client apps on unmanaged iOS devices. \n\nHardens security for policy managed apps and blocks egress of organization data.",
+  "description": "What is it: Configures default behavior for managed client apps on unmanaged iOS devices. Hardens security for policy managed apps and blocks egress of organization data. \n Simeon Notes: Protect your data on users' personal iOS devices. This policy allows data to flow between approved apps, but does not allow users to copy or save data outside of the app. Simeon's baseline applies this policy to all users except those in the Azure AD group, 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy",
```

Comment:
**What does this do?**
Configures the default behavior for managed client apps on unmanaged iOS devices. The baseline allows data to flow between approved apps, but prohibits users from copying or saving data outside of the app except for users in the Azure AD group, 'Baseline - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. 

**Why should you use this?**
 If you want to further protect your data on users' personal iOS devices by blocking egress of organization data.  @metamenos is there a word we could use other than egress?

**What is the end-user impact?**
 iOS device users cannot copy or save data outside of your managed client apps. 

**Learn more**
[App protection policies overview](https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy)

Reply:
@metamenos I updated this one accordingly